utils/exit-on-error.js

```diff
@@ -32,6 +32,7 @@ process.on('exit', () => {
 });
 
 module.exports = (promiseFn) => async (...args) => {
+
```

Comment:
```suggestion
```

Reply:
Thanks! I have no power here and @volovyk-s is the better bet when tagging for a review.